internal/builtins/testData/pkg-with-existing-ctx/in.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: config.kubernetes.io/v1
+kind: ResourceList
+items:
+  - apiVersion: kpt.dev/v1
+    kind: Kptfile
+    metadata:
+      name: order-service
+      annotations:
+        internal.config.kubernetes.io/index: '0'
+        internal.config.kubernetes.io/path: 'Kptfile'
+        internal.config.kubernetes.io/seqindent: 'compact'
+  - apiVersion: v1
+    kind: Namespace
+    metadata:
+      name: example-ns
+      annotations:
+        internal.config.kubernetes.io/index: '0'
+        internal.config.kubernetes.io/path: 'ns.yaml'
+        internal.config.kubernetes.io/seqindent: 'compact'
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: kptfile.kpt.dev
+      annotations:
+        config.kubernetes.io/local-config: "true"
+        internal.config.kubernetes.io/path: 'package-context.yaml'
+    data:
+      name: order-service
```

Comment:
Is this part the generated biultin pkg-context? which means they should not in the `in.yaml` but only in the `out.yaml`?

Reply:
This is testing idempotency of the `gen-pkg-context` function which ensures correct pkg context is generated even if input contains pkg-context.yaml.